src/wizard/javascript/angularjs.md

```diff
@@ -1,121 +1,45 @@
-<script>
-  Raven.config("___PUBLIC_DSN___")
-    .addPlugin(Raven.Plugins.Angular)
-    .install();
-</script>
-```
-
-##### npm
-
-```shell
-npm install raven-js --save
-```
-
-```html
-<script src="/node_modules/angular/angular.js"></script>
-<script src="/node_modules/raven-js/dist/raven.js"></script>
-<script src="/node_modules/raven-js/dist/plugins/angular.js"></script>
-<script>
-  Raven.config("___PUBLIC_DSN___")
-    .addPlugin(Raven.Plugins.Angular)
-    .install();
-</script>
-```
-
-These examples assume that AngularJS is exported globally as _window.angular_. You can alternatively pass a reference to the _angular_ object directly as the second argument to _addPlugin_:
-
-```javascript
-Raven.addPlugin(Raven.Plugins.Angular, angular);
-```
-
-#### Module loaders (CommonJS)
-
-Raven and the Raven AngularJS plugin can be loaded using a module loader like Browserify or Webpack.
-
-```javascript
-var angular = require("angular");
-var Raven = require("raven-js");
-
-Raven.config("___PUBLIC_DSN___")
-  .addPlugin(require("raven-js/plugins/angular"), angular)
-  .install();
-```
-
-Note that when using CommonJS-style imports, you must pass a reference to _angular_ as the second argument to _addPlugin_.
-
-### AngularJS Configuration
-
-Inside your main AngularJS application module, you need to declare _ngRaven_ as a module dependency:
-
-```javascript
-var myApp = angular.module("myApp", [
-  "ngRaven",
-  "ngRoute",
-  "myAppControllers",
-  "myAppFilters",
-]);
+# Using npm
+npm install --save @sentry/browser @sentry/integrations
 ```
 
-#### Module loaders (CommonJS) {#id1}
-
-The raven angular module can be loaded using a module loader like Browserify or Webpack.
+You should `init` the Sentry browser SDK as soon as possible during your application load up, before initializing Angular:
 
 ```javascript
-var angular = require("angular");
-var ngRaven = require("raven-js/plugins/angular").moduleName;
-var ngRoute = require("angular-route");
-
-var myAppFilters = require("./myAppFilters");
-var myAppControllers = require("./myAppControllers");
-var moduleName = "myApp";
-
-angular.module(moduleName, [ngRaven, ngRoute, myAppControllers, myAppFilters]);
-
-module.exports = moduleName;
+import angular from "angular";
+import * as Sentry from "@sentry/browser";
+import { Integrations } from "@sentry/tracing";
+import { Angular as AngularIntegration } from "@sentry/integrations";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  autoSessionTracking: true,
+  integrations: [
+    new AngularIntegration(),
+    new Integrations.BrowserTracing({
+      tracingOrigins: ["localhost", "https://yourserver.io/api"],
+    }),
+  ],
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
+});
+
+// Finally require ngSentry as a dependency in your application module.
+angular.module("yourApplicationModule", ["ngSentry"]);
 ```
-
```

Comment:
We recommend adjusting the value of `tracesSampleRate` in production. Learn more about configuring sampling in our [full documentation](https://docs.sentry.io/platforms/javascript/performance/sampling/).

Reply:
We're also adding this to the end of each wizard (after the verification example):
If you're new to Sentry, use the email alert to access your account and complete a product tour.

If you're an existing user and have disabled alerts, you won't receive this email.